### ISSUE TITLE
Better control over experiments

### ISSUE THREAD

**Issue by jbingel** [j6mes/springer2018-hatespeech-bridging-gaps#20]
This will be a bit of work... Hyperparameters are all hardcoded as of now. Would be good to have:

- dynamic graph creation (number of hidden layers, hidden layer width, optimizers...)
- setting learning-related hyperparams externally (argparsing learning rate, optimizer, graph structure, early stopping patience, ...)
- ...
